### PR TITLE
support: gw7740 vrt tag

### PR DESCRIPTION
**Pull request by mudana-grune**
https://youtrack.weseek.co.jp/issue/GW-7740

### CODE REVIEW

File: packages/app/test/cypress/integration/20-basic-features/use-tools.spec.ts [hunk 1]
```diff
@@ -141,3 +141,167 @@ context('Page Accessories Modal', () => {
...
+      cy.get('button.dropdown-toggle').first().click({force: true});
+      cy.get('.dropdown-menu-right').should('be.visible');
+      cy.get('.dropdown-menu-right > button:nth-child(3)').click({force: true});
+    });
+    cy.getByTestid('search-result-base').should('be.visible');
+    cy.getByTestid('search-result-list').should('be.visible');
+    cy.getByTestid('search-result-content').should('be.visible');
+    cy.screenshot(`${ssPrefix}by-last-update-date`);
+  });
+});
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/use-tools.spec.ts:307]: I think this part of tests should be in `packages/app/test/cypress/integration/30-search/search.spec.ts`

File: packages/app/test/cypress/integration/20-basic-features/use-tools.spec.ts [hunk 2]
```diff
@@ -141,3 +141,167 @@ context('Page Accessories Modal', () => {
...
+    cy.fixture("user-admin.json").then(user => {
+      cy.login(user.username, user.password);
+    });
+  });
+
+  it('Successfully add new tag', () => {
+    const ssPrefix = 'tag-operations-add-new-tag-'
+    const tag = 'we';
+    cy.visit('/');
+    cy.screenshot(`${ssPrefix}click-plus-button`, {capture: 'viewport'});
```
Comment (Yohei-Shiina) [inline packages/app/test/cypress/integration/20-basic-features/use-tools.spec.ts:158]: I don't think you need this line

File: packages/app/src/components/PageRenameModal.tsx [hunk 3]
```diff
@@ -347,6 +348,7 @@ const PageRenameModal = (): JSX.Element => {
           className="btn btn-primary"
           onClick={rename}
           disabled={submitButtonDisabled}
+          data-testid="rename-page-button"
```
Comment (Yohei-Shiina) [inline packages/app/src/components/PageRenameModal.tsx:351]: Ignore and Resolve this comment if you are still working on this PR.

If not, `data-testid="new-page-name-input"` and `data-testid="rename-page-button"` are not used

Comment (Yohei-Shiina) [reply packages/app/src/components/PageRenameModal.tsx:351]: You are not using `new-page-name-input` at line 255

Please review your own PR before you resolve the commnet.

Comment (mudana-grune) [reply packages/app/src/components/PageRenameModal.tsx]: Just removed data-testid in PageRenameModal

**Pull request merged**